fhir/com.b2international.snowowl.fhir.rest/src/com/b2international/snowowl/fhir/rest/MetadataController.java

```diff
@@ -0,0 +1,387 @@
+			//properties = @ExtensionProperty(name = "type", value = "String")), 
+			//description = "Canonical URL of the value set")
+			
+			HttpServletRequest request) {
+		
+		if (capabilityStatement == null) {
+			initCache(request);
+		}
+		return Promise.immediate(capabilityStatement);
+	}
+	
+	/**
+	 * Returns the {@link OperationDefinition} for a given operation.
+	 * @param operation
+	 * @return
+	 */
+	@Operation(
+			summary="Retrieve an operation definition by its name", 
+			description="Retrieves an operation definition by its compound name (resource$operation)."
+	)
+	@ApiResponses({
+		@ApiResponse(responseCode = "200", description = "OK"),
+		@ApiResponse(responseCode = "404", description = "Operation definition not found")
+	})
+	@GetMapping(value="/OperationDefinition/{operation}")
+	public Promise<OperationDefinition> operationDefinition(
+			@PathVariable(value = "operation") final String operation,
+			HttpServletRequest request) {
+		
+		if (capabilityStatement == null) {
+			initCache(request);
+		}
+		
+		if (!operationMap.containsKey(operation)) {
+			throw new NotFoundException("OperationDefinition", operation);
+		}
+		
+		//get the definition from the cached map
+		OperationDefinition operationDefinition = operationMap.get(operation);
+		return Promise.immediate(operationDefinition);
+	}
+	
+	
+	private synchronized void initCache(final HttpServletRequest request) {
+		
+		OpenAPI openAPI = openApiResource.getOpenApi();
+		
+		Paths paths = openAPI.getPaths();
+		
+		Set<String> fhirOperationKeys = paths.keySet().stream()
+				.filter(k -> k.startsWith("/fhir"))
+				.filter(k -> k.contains("$"))
+				.collect(Collectors.toSet());
+		
+		Collection<Resource> resources = collectResources(openAPI);
+		
+		Builder restBuilder = Rest.builder()
+				.mode(RestfulCapabilityMode.SERVER)
+				.resources(resources);
+		
+		for (String fhirKey : fhirOperationKeys) {
+			
+			PathItem pathItem = paths.get(fhirKey);
+			
+			//Skip non-GET type of operations
+			if (pathItem.getGet() == null) continue;
+			
+			OperationDefinition operationDefinition = buildOperationDefinition(fhirKey, pathItem);
+			
+			operationDefinition.getResources().stream().forEach(c -> {
+				operationMap.put(c.getCodeValue() + operationDefinition.getName(), operationDefinition);
+				restBuilder.addOperation(com.b2international.snowowl.fhir.core.model.capabilitystatement.Operation.builder()
+						.name(operationDefinition.getName())
+						.definition(buildOperationUrl(request, c, operationDefinition))
+						.build());
+			});
+		}
+		
+		capabilityStatement = CapabilityStatement.builder()
+				.name("FHIR Capability statement for Snow Owl© Terminology Server")
+				.status(PublicationStatus.ACTIVE)
+				.experimental(false)
+				.kind(CapabilityStatementKind.INSTANCE.getCode())
+				.addInstantiate(new Uri("http://hl7.org/fhir/CapabilityStatement/terminology-server"))
+				.software(Software.builder()
+						.name("Snow Owl©")
+						.version("8.0.0")
```

Comment:
Version can be extracted via `Platform.getBundle(CoreActivator.PLUGIN_ID).getVersion()`.